spec/factories/item.rb

```diff
@@ -2,7 +2,7 @@
   factory :random_item, class: Item do
     sequence(:name)   { Faker::Commerce.product_name }
     description       { Faker::Hipster.sentence(word_count: 5) }
-    price             { Faker::Commerce.price }
+    price             { rand(0.01..100.99).to_s } #Faker::Commerce.price price above 0
```

Comment:
I think you can use something like this:

{ Faker::Commerce.price(range = 0.1..100.0, as_string = false) }

I haven't used this, so I'd test to make sure I'm not fucking it up 😄